### PR TITLE
Add route53 and certificate entries

### PR DESCRIPTION
**Pull request by archferns**
Add a custom domain name for UAT environment of the [court-data-adaptor](https://github.com/ministryofjustice/laa-court-data-adaptor/)

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-uat/resources/route53.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "laa_crime_apps_team_route53_zone" {
+  name = "uat.court-data-adaptor.service.justice.gov.uk"
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-uat/resources/route53.tf]: Can you please update the zone name to some thing like
`court-data-adaptor.service.justice.gov.uk` so later you can use sub-domains by naming your ingress as  `uat.court-data-adaptor.service.justice.gov.uk`

Comment (archferns) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-uat/resources/route53.tf]: updated to use a higher level name.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-prod/resources/route53.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    environment-name       = "production"
+    owner                  = "LAA Crime Apps Team: laa-crime-apps@digital.justice.gov.uk"
+    infrastructure-support = "laa-crime-apps@digital.justice.gov.uk"
+  }
+}
+
+resource "kubernetes_secret" "laa_crime_apps_route53_zone_sec" {
+  metadata {
+    name      = "laa-crime-apps-route53-zone-output"
+    namespace = "laa-court-data-adaptor-uat"
```
Comment (vijay-veeranki-moj) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-prod/resources/route53.tf]: can you please change the namespace name to `laa-court-data-adaptor-prod`

Comment (archferns) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-court-data-adaptor-prod/resources/route53.tf]: Thanks, updated!

**Pull request merged**